packages/app/obojobo-express/models/draft.js

```diff
@@ -72,6 +72,20 @@ class Draft {
 		return draftNode
 	}
 
+	// returns a boolean
+	static async userHasPermissionToDraft(userId, draftId) {
+		const result = await db.oneOrNone(
+			`SELECT user_id
+			FROM drafts
```

Comment:
Is this correct, or should it be pulling from the map_user_to_draft table?

Reply:
Here - yes. Keep in mind this PR is intended to keep `obojobo-express` decoupled from other packages.

From the perspective of obojobo-express, the only owner is the creator and needs that to operate without repository. obojobo-repository implements the concept of multiple owners (db tables, queries, etc.). The promiseProvider allows these decoupled libraries to manage permissions.

When the following is executed:
```
const hasPermResults = await callPromiseProviders('userHasPermissionToDraft', args, true)
```

The promise provider will call each provider in series. First obojobo-express's then any others.  In our current setup, obojobo-repository has a listener that will check the `repository_map_user_to_draft` table.